theano/gpuarray/elemwise.py

```diff
@@ -41,48 +41,6 @@ def get_scal(dt):
     return scalar.get_scalar_type(dt)
 
 
-def max_inputs_to_GpuElemwise(node_or_outputs):
```

Comment:
All the change to this PR is probably due to a bad rebase/merge. Make sure they don't appear in this PR.